state/txn_test.go

```diff
@@ -100,7 +100,8 @@ func buildMockPreState(p *PreState) (*Account, *mockSnapshot) {
 	account := &Account{
 		Nonce:   p.Nonce,
 		Balance: big.NewInt(int64(p.Balance)),
-		Root:    root,
+		// TODO Mock the Staked balance
+		Root: root,
```

Comment:
This `TODO` is very persistent 😄 
```suggestion
		Root: root,
```

Reply:
Fixed 👍